### PR TITLE
Fix multiple get_enable_ipex function error

### PR DESCRIPTION
**Pull request by Romanticoseu**
## Description

Fix multiple get_enable_ipex function error

### CODE REVIEW

File: python/llm/src/bigdl/llm/transformers/__init__.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #
 
 
-from .convert import ggml_convert_low_bit, get_enable_ipex
+from .convert import ggml_convert_low_bit, get_enable_ipex, get_enable_ipex_low_bit
```
Comment (glorysdj) [inline python/llm/src/bigdl/llm/transformers/__init__.py:18]: why multiple get_enable_ipex?
@xiangyuT we should simplify this

File: python/llm/src/bigdl/llm/transformers/loader.py [hunk 2]
```diff
@@ -65,7 +65,7 @@ def load_model(
     # Load tokenizer
     tokenizer = tokenizer_cls.from_pretrained(model_path, trust_remote_code=True)
     model = model_cls.from_pretrained(model_path, **model_kwargs)
-    if not get_enable_ipex(low_bit):
+    if not get_enable_ipex_low_bit(low_bit):
```
Comment (xiangyuT) [inline python/llm/src/bigdl/llm/transformers/loader.py:68]: we can remove function `get_enable_ipex_low_bit`, just check by `get_enable_ipex()`. IPEX-optimized models always don't need `.eval()`

**Review by glorysdj — APPROVED** [review]
LGTM

**Review by xiangyuT — APPROVED** [review]
LGTM

**Pull request merged**